### PR TITLE
feat: Marketplace data import

### PR DESCRIPTION
**Pull request by kiwi2021**
## Summary of Changes
Data for the aircraft's marketplace integration.

## Screenshots (if necessary)
**Thumbnail**
Thumbnail

**Main Background**
Background 1
Background 2
Background 3
Background 4
Background 5

**Description and History**
Description History

**Specification**
Specification

**Image Slider**
Image Slider 1
Image Slider 2
Image Slider 3
Image Slider 4
Image Slider 5
Image Slider 6
Image Slider 7
Image Slider 8
Image Slider 9

**Version History**
Version History

## References
All aircraft specification was obtained via the official Airbus website: [Here](https://www.airbus.com/aircraft/passenger-aircraft/a320-family/a320neo.html)

And in-line with specifications outlines in #3474 

Discord username (if different from GitHub): kiwi2022#0001

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: .github/CHANGELOG.md [hunk 1]
```diff
@@ -238,6 +238,7 @@
 1. [TEXTURE] Updated glare shield leather, flyPad holder, tray table, overhead and push-button textures - @MoreRightRudder (Mike)
 1. [MCDU] Progress page shows current flight phase - @Username23-23 (NONAmr2433 #8777)
 1. [TEXTURE] Upscaled the external airframe decals to go nicely with the upscaled livery. @FoxinTale (Aubrey)
+1. [MISC] Added Marketplace Data. @kiwi2021 (kiwi2022#0001)
```
Comment (pepperoni505) [inline .github/CHANGELOG.md:241]:
```suggestion
1. [TEXTURE] Upscaled the external airframe decals to go nicely with the upscaled livery - @FoxinTale (Aubrey)
1. [MISC] Added Marketplace Data -  @kiwi2021 (kiwi2022#0001)
```

File: flybywire-aircraft-a320-neo/Business.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "PriceInUSD": 0,
+  "ThirdPartyShortName": "flybywiresimulations",
```
Comment (pepperoni505) [inline flybywire-aircraft-a320-neo/Business.json:3]:
```suggestion
  "ThirdPartyShortName": "flybywire",
```
Should change to that to match our package name

File: flybywire-aircraft-a320-neo/layout.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+  "content": []
+}
```
Comment (pepperoni505) [inline flybywire-aircraft-a320-neo/layout.json:3]: Is this file needed?

File: flybywire-aircraft-a320-neo/manifest.json [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  "dependencies": [],
+  "content_type": "AIRCRAFT",
+  "title": "A32NX",
+  "manufacturer": "Airbus",
+  "creator": "FlyByWire Simulations",
+  "package_version": "0.6.0",
+  "minimum_game_version": "1.14.6",
+  "release_notes": {
+    "neutral": {
+      "LastUpdate": "VERSION 0.6.0 RELEASED APRIL 6, 2021\\nInitial Release onto Marketplace",
```
Comment (pepperoni505) [inline flybywire-aircraft-a320-neo/manifest.json:11]: Should probably change this, since 0.6.0 hasn't even been released yet

File: flybywire-aircraft-a320-neo/layout.json (hunk 3 above)
Comment (kiwi2021) [reply flybywire-aircraft-a320-neo/layout.json:3]: It was generated alongside the other two .json files. I would hazard a guess to say that we do not need this file, considering the builder in-sim would have populated this if we would require.

Could you or someone else with the CRJ check to see if this file is included? (Or any other marketplace aircraft)

File: flybywire-aircraft-a320-neo/manifest.json (hunk 4 above)
Comment (kiwi2021) [reply flybywire-aircraft-a320-neo/manifest.json:11]: I thought the first version going onto the marketplace was V0.6.0

File: flybywire-aircraft-a320-neo/layout.json (hunk 3 above)
Comment (pepperoni505) [reply flybywire-aircraft-a320-neo/layout.json:3]: CRJ has it - so I guess we should keep it

File: flybywire-aircraft-a320-neo/manifest.json (hunk 4 above)
Comment (pepperoni505) [reply flybywire-aircraft-a320-neo/manifest.json:11]: Yes, however we are not planning on releasing it on April 6th. Also, this file can be deleted. Similar to layout.json, we also generate ours. I do recommend editing manifest-base.json in the root of the project to add the `LastUpdate` field

Comment (kiwi2021) [reply flybywire-aircraft-a320-neo/manifest.json:11]: LastUpdate exists in the manufest-base.json, it is just not populated. We could populate this however would be best when we know the definitive release date. I could add `"LastUpdate": "VERSION 0.6.0 RELEASED APRIL 6, 2021\\nInitial Release onto Marketplace",` as a placeholder if we wanted to ensure compatibility with the Marketplace?

Comment (pepperoni505) [reply flybywire-aircraft-a320-neo/manifest.json:11]: Hmm. Maybe we could do that without the released part. I don't think it requires a release date

Comment (kiwi2021) [reply flybywire-aircraft-a320-neo/manifest.json:11]: This is how it shows on the 'Version History'

image

Comment (kiwi2021) [reply flybywire-aircraft-a320-neo/manifest.json:11]: This is how it looks without the date. 
image

Comment (pepperoni505) [reply flybywire-aircraft-a320-neo/manifest.json:11]: Maybe `VERSION 0.6.0\\nInitial Release onto Marketplace`

Comment (kiwi2021) [reply flybywire-aircraft-a320-neo/manifest.json:11]: LGTM
image

Comment (pepperoni505) [reply flybywire-aircraft-a320-neo/manifest.json:11]: Yep, that looks good

Comment (pepperoni505) [reply flybywire-aircraft-a320-neo/manifest.json:11]: Don't forget to delete this file

**Review by pepperoni505 — APPROVED** [review]
LGTM

File: flybywire-aircraft-a320-neo/Business.json [hunk 5]
```diff
@@ -0,0 +1,6 @@
+{
+  "PriceInUSD": 0,
+  "ThirdPartyShortName": "flybywire",
+  "AccountSellerId": "",
```
[the remark this answers is not in the document]
Comment (kiwi2021) [reply flybywire-aircraft-a320-neo/Business.json:4]: The seller ID will be given by Microsoft

File: .github/CHANGELOG.md [hunk 6]
```diff
@@ -240,7 +240,8 @@
 1. [TEXTURE] Fixed decal clipping issues on the overhead fuel panel - @ImenesFBW (Imenes)
 1. [TEXTURE] Updated glare shield leather, flyPad holder, tray table, overhead and push-button textures - @MoreRightRudder (Mike)
 1. [MCDU] Progress page shows current flight phase - @Username23-23 (NONAmr2433 #8777)
-1. [TEXTURE] Upscaled the external airframe decals to go nicely with the upscaled livery. @FoxinTale (Aubrey)
+1. [TEXTURE] Upscaled the external airframe decals to go nicely with the upscaled livery - @FoxinTale (Aubrey)
+1. [MISC] Added Marketplace Data -  @kiwi2021 (kiwi2022#0001)
```
Comment (TansyEliot) [inline .github/CHANGELOG.md:244]:
```suggestion
1. [MISC] Added Marketplace Data - @kiwi2021 (kiwi2022#0001)
```

**Review by TansyEliot — CHANGES_REQUESTED** [review]
Hey, you just forgot an extra space between "-" and "@kiwi2021", can you remove it posteriorly?

File: flybywire-aircraft-a320-neo/MarketplaceData/flybywiresimulations-a32nx/Marketplace.json [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+                "name": "MAX ALTITUDE",
+                "valueStr": "39,698 Feet"
+              },
+              {
+                "name": "RANGE",
+                "valueStr": "3,401 Nautical Miles"
+              },
+              {
+                "name": "EMPTY WEIGHT",
+                "valueStr": "44,300 KG"
```
[the remark this answers is not in the document]
Comment (kiwi2021) [reply flybywire-aircraft-a320-neo/MarketplaceData/flybywiresimulations-a32nx/Marketplace.json]: As we are intending to simulate these values, I have changed the specs here to match so we don't have conflicting information.

File: flybywire-aircraft-a320-neo/MarketplaceData/flybywiresimulations-a32nx/Marketplace.json [hunk 8]
```diff
@@ -0,0 +1,105 @@
...
+              },
+              {
+                "name": "MAX WEIGHT",
+                "valueStr": "79,000 KG"
+              },
+              {
+                "name": "FUEL CAPACITY",
+                "valueStr": "36,730 Litres"
```
[the remark this answers is not in the document]
Comment (kiwi2021) [reply flybywire-aircraft-a320-neo/MarketplaceData/flybywiresimulations-a32nx/Marketplace.json]: Same again, I have matched these values now.

File: flybywire-aircraft-a320-neo/MarketplaceData/flybywiresimulations-a32nx/Marketplace.json [hunk 9]
```diff
@@ -0,0 +1,105 @@
+{
+  "market-data": {
+    "Neutral": {
+      "creator": "FlyByWire Simulations",
+      "title": "A32NX",
```
[the remark this answers is not in the document]
Comment (pepperoni505) [reply flybywire-aircraft-a320-neo/MarketplaceData/flybywiresimulations-a32nx/Marketplace.json:5]: Yes, we probably will

**Review by tracernz — APPROVED** [review]

**Comment by kiwi2021** [comment]
Label applied whilst I look into directory requirements. Will be looked at tomorrow, 7th April.

**Review by davidwalschots — APPROVED** [review]

**Review by pepperoni505 — APPROVED** [review]
LGTM

**Comment by wpine215** [comment]
Does this need traditional QA testing?

**Comment by pepperoni505** [comment]
> Does this need traditional QA testing?

No, AFAIK there is no way for QA to test this unless they open a dummy project in dev mode and copy this over

**Review by Benjozork — APPROVED** [review]
LGTM

**Comment by Watsi01** [comment]
Well done.

**Comment by kiwi2021** [comment]
> Does this need traditional QA testing?

Adding to what Pep said. Pep has a copy of the files too including the XML. I can pass them over to yourself also. That way we always have one available should we need to make changes requiring testing.
